views/index.hbs

```diff
@@ -1,3 +1,7 @@
 <div>
+<img src="/public/images/beer.png" alt="beer">
 
+<ul class = "beerButtons">
+            <li> <a class="" href="/views/beers.hbs"> Check the Beers! </a> </li>
```

Comment:
The `path` that you need to specify in the `href` is the `url` path that you specified in the `app.js` and not the path to the `html` file:

```
            <li> <a class="" href="/beers"> Check the Beers! </a> </li>
            <li> <a class="" href="/random-beer"> Check a Random Beer </a> </li>
```